xarray/core/computation.py

```diff
@@ -646,6 +646,8 @@ def apply_variable_ufunc(
 
             if dask_gufunc_kwargs is None:
                 dask_gufunc_kwargs = {}
+            else:
+                dask_gufunc_kwargs = dask_gufunc_kwargs.copy()
```

Comment:
Very narrow point, but we should probably use `copy.copy`, since not all `Mapping`s have a `copy` method.

Reply:
In this case it is a `dict` which has `copy` method. If it should be changed anyhow to use `copy.copy` please let me know.

Reply:
You're right, we assume it's a dict elsewhere already